src/patternfly/components/Tabs/examples/tabs-nav.hbs

```diff
@@ -59,22 +59,22 @@
 
 <br><br>
 
-{{#> tabs tabs--id="primary-nav" tabs--type="nav" tabs--attribute='aria-label="Local"'}}
+{{#> tabs tabs--id="primary-nav" tabs--type="nav" tabs--attribute='aria-label="Local"' tabs-button--type="a"}}
```

Comment:
If my previous comment applies (if tabs = nav, tabs button = a) I wonder if tabs--type is enough and we can assign the tabs-button--type through there